erpnext/stock/doctype/item/item.json

```diff
@@ -57,6 +57,7 @@
   "quotation_validity_days",
   "column_break_35",
   "vehicle_colors",
+  "vehicle_interior",
```

Comment:
fieldname should be plural for table fields

Reply:
Should I change it everywhere?

Reply:
Do not change, I will change it myself